third_party/bigframes_vendored/pandas/core/frame.py

```diff
@@ -467,6 +468,9 @@ def to_gbq(
                 clustering order within the Index/DataFrame columns follows the order
                 specified in `clustering_columns`.
 
+            labels (dict[str, str], default None):
+                Specifies table labels within BigQuery
```

Comment:
nit, (but since this goes in the public doc) should end with the period `.`

Reply:
done in https://github.com/googleapis/python-bigquery-dataframes/pull/968!